### PR TITLE
Avoid suggesting plural for singular words

### LINKED ISSUE #1427: Incorrect suggestion: resoultion -> resolutions
However, it should be more like resoultion -> resolution (singular)

### PR DESCRIPTION
**Pull request by tbm**
Fixes #1427

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -15749,7 +15749,7 @@ ist->is, it, its, it's, list,
 istalling->installing
 istance->instance
 istead->instead
-istener->listeners
+istener->listener
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:15752]: Can we add the plural version too please:
`isteners->listeners`
Also probably the -ed and -ing versions (the latter certainly exists in grep.app).

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11235,7 +11235,7 @@ exeptions->exceptions
 exeqution->execution
 exerbate->exacerbate
 exerbated->exacerbated
-exerciese->exercises
+exerciese->exercise
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:11238]: Can we add the plural, -ed and -ing variations of this typo too.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -11235,7 +11235,7 @@ exeptions->exceptions
...
 exercize->exercise
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:11239]: Would you mind doing the same for this one as well?

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (tbm) [reply codespell_lib/data/dictionary.txt:15752]: @peternewman I'd be happy to submit those changes but I think it should be a different PR (or at least a separate commit). Do you agree?

Comment (peternewman) [reply codespell_lib/data/dictionary.txt:15752]: Yeah another commit is fine (and my preference as GitHub plays better then).

Comment (tbm) [reply codespell_lib/data/dictionary.txt:15756]: done in second commit

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (tbm) [reply codespell_lib/data/dictionary.txt:11242]: done in second commit

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (tbm) [reply codespell_lib/data/dictionary.txt:11238]: done in second commit

**Comment by peternewman** [comment]
Thanks @tbm !

**Pull request merged**